src/main/java/net/neoforged/neoforge/event/EventHooks.java

```diff
@@ -183,9 +184,9 @@ public static NeighborNotifyEvent onNeighborNotify(Level level, BlockPos pos, Bl
         return event;
     }
 
-    public static boolean doPlayerHarvestCheck(Player player, BlockState state, boolean success) {
-        PlayerEvent.HarvestCheck event = new PlayerEvent.HarvestCheck(player, state, success);
-        NeoForge.EVENT_BUS.post(event);
+    public static boolean doPlayerHarvestCheck(Player player, BlockState state, BlockGetter level, BlockPos pos) {
+        boolean vanillaValue = player.hasCorrectToolForDrops(state);
```

Comment:
IMO we should document why the overload with extra parameters is not called here (because this is actually the implementation of it, but it's not obvious from just looking at the one file).

Reply:
What exactly do you mean? Just add a comment along the lines of "this calls the deprecated method because the non-deprecated one calls this in return"?

Reply:
Yes